README.md

```diff
@@ -16,3 +16,11 @@ You can also configure the port (with `-p`) and root directory (with
 ```bash
 java duckling.jar -p $(YOUR_FAVORITE_PORT) -d $(YOUR_FAVORITE_DIR)
 ```
+
+## A small overview
+
+Right now Duckling doesn't support routing at all.  It will display some HTML as expected, provide a small navigable directory listing, and respond with a 404 to missing content.
+
+## Known bugs
+
+* Deep traversal of directories is currently not supported.  This is getting punted on as of release 0.0.1 in the interests of getting a more thoroughly tested `FolderContents` and usable responder in play, vs. upgrading the current one further (which is moot, because it's serving its purpose as-is).
```

Comment:
Gotcha - this is a good one for me to internalize, so I'll make my first amendment based on it.  Seems like I have some notions to act on:

1. Make an issue to track a problem.
2. Try to keep issues resolved quickly / at 0.
3. Don't introduce known bugs.